app/events.py

```diff
@@ -91,6 +93,32 @@ def getEventObj(event):
 
 nameAddressRegex = re.compile('^([^0-9]*)([0-9]*.*)')
 
+def fetchEventsFromLocation(latlong, maxResults, radius):
+    params = { 'app_key': eventfulkey,
+               'units': 'mi',
+               'date': 'Today',
+               'sort_order': 'relevance',
+               'location': latlong,
+               'page_size': maxResults,
+               'within': radius }
+
+    r = requests.get(EVENTFUL_URL, params)
+    eventList = r.json()['events']['event']
+    return eventList
+
+def getEventfulEventObj(event):
+    location = { 'lat': event['latitude'], 'lng': event['longitude'] }
```

Comment:
Gotcha - the gcal API returns 'lng' so instead of switching it over, I used the response as-is - this is to remain consistent with the previous pull request. Since the client is already consuming the other data, I think it'll be better to use 'lon' instead - I'll add one more commit to make the Events code consistent.